dim/lib/dim/commands/export.rb

```diff
@@ -3,6 +3,7 @@
 
 require_relative '../globals'
 require_relative '../exporter/exporterInterface'
+require_relative '../exporter/raw_rst'
```

Comment:
Alternatively, could be integrated into the rst exporter with some "ifs"...

Reply:
What do you think about instantiating either `RST` or `RawRST` in accordance? The problem that I currently see is that there is no way to pass additional options to the exporter, it only takes the format and the output directory, so how would it know wether to use `RST` or `RawRST`?